package.json

```diff
@@ -7,10 +7,13 @@
   "bugs": "https://github.com/facebook/relay/issues",
   "repository": "facebook/relay",
   "scripts": {
-    "build": "gulp",
+    "build": "gulp dist",
```

Comment:
This will now perform incremental builds, so running `yarn install` after a `git pull` will be faster.